app/AppCache.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class AppCache extends HttpCache
+{
+  class HogeClass {
+    Class HogeHogeClass {
+    
+    }
+  }
+$ddddddddddddddddddddddddddddddddddddddddddddddddddd= '';
+
+}
```

Comment:
__[code_sniffer]__
* The closing brace for the class must go on the next line after the body